### PR TITLE
Update restream.md

### PR DESCRIPTION
**Pull request by lpdescamps**
show the code for streaming birdseye

### CODE REVIEW

File: docs/docs/configuration/restream.md [hunk 1]
```diff
@@ -18,6 +18,13 @@ You can access the go2rtc webUI at `http://frigate_ip:5000/live/webrtc` which ca
 ### Birdseye Restream
 
 Birdseye RTSP restream can be enabled at `birdseye -> restream` and accessed at `rtsp://<frigate_host>:8554/birdseye`. Enabling the restream will cause birdseye to run 24/7 which may increase CPU usage somewhat.
+```
+birdseye:
+  enabled: True
+  mode: continuous
```
Comment (NickM-27) [inline docs/docs/configuration/restream.md:24]: This isn't required for restream so best to not include

File: docs/docs/configuration/restream.md [hunk 2]
```diff
@@ -18,6 +18,13 @@ You can access the go2rtc webUI at `http://frigate_ip:5000/live/webrtc` which ca
 ### Birdseye Restream
 
 Birdseye RTSP restream can be enabled at `birdseye -> restream` and accessed at `rtsp://<frigate_host>:8554/birdseye`. Enabling the restream will cause birdseye to run 24/7 which may increase CPU usage somewhat.
```
Comment (NickM-27) [inline docs/docs/configuration/restream.md:20]:
```suggestion
 Birdseye RTSP restream can be accessed at `rtsp://<frigate_host>:8554/birdseye`. Enabling the restream will cause birdseye to run 24/7 which may increase CPU usage somewhat.

```

**Review by NickM-27 — CHANGES_REQUESTED** [review]

File: docs/docs/configuration/restream.md [hunk 3]
```diff
@@ -17,7 +17,15 @@ You can access the go2rtc webUI at `http://frigate_ip:5000/live/webrtc` which ca
 
 ### Birdseye Restream
 
-Birdseye RTSP restream can be enabled at `birdseye -> restream` and accessed at `rtsp://<frigate_host>:8554/birdseye`. Enabling the restream will cause birdseye to run 24/7 which may increase CPU usage somewhat.
+ Birdseye RTSP restream can be accessed at `rtsp://<frigate_host>:8554/birdseye`. Enabling the restream will cause birdseye to run 24/7 which may increase CPU usage somewhat.
+
+```
+birdseye:
+  enabled: True
+  mode: continuous
```
Comment (NickM-27) [inline docs/docs/configuration/restream.md:25]:
```suggestion
```

**Review by NickM-27 — APPROVED** [review]

**Comment by NickM-27** [comment]
@blakeblackshear looks like the tests for master are failing for some reason, guess something is out of date. not sure if it makes sense to merge as is anyway knowing that this is fixed in dev or if these PRs should be updated to target dev

**Comment by blakeblackshear** [comment]
I would just update the PR to dev

**Pull request closed without merging**